identity/authentication.go

```diff
@@ -145,6 +145,32 @@ func splitToken(token string) (tokenObj tokenObject) {
 	return tokenObj
 }
 
+func createUserAuthRequest(url string, userAuthToken string) (*http.Request, error) {
```

Comment:
Only minor but think this function is missing an annotation, although it is a self-explaining function name, other functions in the file have done this, so think we should do this here too.